packages/contracts-bedrock/README.md

```diff
@@ -56,11 +56,13 @@ npm install @eth-optimism/contracts-bedrock
 
 We work on this repository with a combination of [Hardhat](https://hardhat.org) and [Foundry](https://getfoundry.sh/).
 
-1. Install Foundry by following [the instructions located here](https://getfoundry.sh/).
-   A specific version must be used.
-   ```shell
-   foundryup -C 3b1129b5bc43ba22a9bcf4e4323c5a9df0023140
-   ```
+1. Install the correct version of foundry:
+
+```shell
+curl -L https://foundry.paradigm.xyz | bash
+yarn update:foundry
+```
+
```

Comment:
🔥 🔥 🔥

Reply:
Maybe we can just add `curl -L https://foundry.paradigm.xyz | bash` as a `yarn install:foundry` target as well?

Reply:
Makes this really idiomatic then with a `yarn install:foundry && yarn update:foundry`

Reply:
or even have `yarn install:foundry` target `yarn update:foundry` as well so you can just run a `yarn install:foundry`